mod/group_tools/views/default/js/group_tools/mail.js

```diff
@@ -28,6 +28,14 @@ elgg.group_tools.mail_clear_members = function() {
 	$('#group_tools_mail_member_selection input[name="user_guids[]"]:checked').each(function() {
 		$(this).removeAttr('checked');
 	});
+    
+    	$('#group_tools_mail_member_selection input[name="select_all"]:checked').each(function() {
+		$(this).removeAttr('checked');
+	});
+
+		$('#group_tools_mail_member_selection input[name="select_all"]:checked').each(function() {
```

Comment:
Is $('#group_tools_mail_member_selection input[name="select_all"]:checked').each(function() {... supposed to be duplicated?